manuscript/developing/02_development_server.md

```diff
@@ -2,46 +2,54 @@
 
 When developing a frontend without any special tooling, you often end up having to refresh the browser to see changes. Given this gets annoying fast, there's tooling to remedy the problem.
 
-The first tools in the market were [LiveReload](http://livereload.com/) and [Browsersync](http://www.browsersync.io/). The point of either is to allow refreshing the browser automatically as you develop. They also pick up CSS changes and apply the new style without a hard refresh that loses the state of the browser.
+The first tools on the market were [LiveReload](http://livereload.com/) and [Browsersync](http://www.browsersync.io/). The point of either is to allow refreshing the browser automatically as you develop. They also pick up CSS changes and apply the new style without a hard refresh that loses the state of the browser.
 
 It's possible to setup Browsersync to work with webpack through [browser-sync-webpack-plugin](https://www.npmjs.com/package/browser-sync-webpack-plugin), but webpack has more tricks in store in the form of a `watch` mode, and a development server.
 
 ## Webpack `watch` mode
 
-Webpack implements a `watch` mode that operates against the project files bundled by webpack. You can activate it by passing the `--watch` to webpack. Example: `npm run build -- --watch`. After this, any change made to a file captured by a webpack will trigger a rebuild.
+Webpack’s `watch` mode rebuilds the bundle on any change of the project files. To activate it, pass the `--watch` switch to webpack:
+
+```bash
+npm run build -- --watch
+```
 
 Although this solves the problem of recompiling your source on change, it does nothing on the frontend side and browser updates. That's where further solutions are required.
 
 {pagebreak}
 
 ## **webpack-dev-server**
 
-[webpack-dev-server](https://www.npmjs.com/package/webpack-dev-server) (WDS) is the officially maintained solution for webpack. WDS is a development server running **in-memory**, meaning the bundle contents aren't written out to files but stored in memory. The distinction is vital when trying to debug code and styles.
+[webpack-dev-server](https://www.npmjs.com/package/webpack-dev-server) (WDS) is the officially maintained development server running **in-memory**, meaning the bundle contents isn't written out to files but stored in memory. The distinction is vital when trying to debug code and styles.
```

Comment:
Likely we should retain `aren't` as `contents` is a plural. Reference: https://english.stackexchange.com/questions/50851/the-contents-are-or-the-contents-is .

Reply:
English is weird 🤷‍♂️